tests/wallet/test_singleton_lifecycle_fast.py

```diff
@@ -33,6 +33,7 @@
 POOL_REWARD_PREFIX_MAINNET = bytes32.fromhex("ccd5bb71183532bff220ba46c268991a00000000000000000000000000000000")
 
 MAX_BLOCK_COST_CLVM = int(1e18)
+COST_PER_BYTE = int(12000)
```

Comment:
Are there any tests in here that could depend on the value of COST_PER_BYTE (and be rejected because of too-high cost?)

Reply:
I can't think of any

Reply:
12000 is the mainnet cost per byte, so it's not too high. But I suppose there could be tests to ensure we fail correctly if this was a larger number. I don't think such test belongs here. `get_name_puzzle_conditions()` isn't really the subject of this test. That said, I think there are a few negative test cases missing